server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -209,3 +213,30 @@ func isRootModified(root terraform.Root, modifiedFiles []string) (bool, error) {
 
 	return false, nil
 }
+
+// emits metrics for past 12 weeks in less than_x_days & more_than_x_days format
+func (r *Runner) emitOpenPRAgeMetrics(ctx workflow.Context, prs []github.PullRequest, numWeeks int) {
+	// track open PRs older than x weeks
+	moreThanXWeeksOld := make([]int, numWeeks)
+	for _, pr := range prs {
+		for i := 0; i < numWeeks; i++ {
+			if isOlderThanXWeeks(ctx, i+1, pr) {
+				moreThanXWeeksOld[i]++
+			}
+		}
+	}
+
+	// emit metrics
+	for i := range moreThanXWeeksOld {
+		moreThan := moreThanXWeeksOld[i]
+		lessThan := len(prs) - moreThan
+		days := (i + 1) * 7
+		r.Scope.SubScope("open_prs").Counter(fmt.Sprintf("more_than_%d_days", days)).Inc(int64(moreThan))
```

Comment:
also, why not just leave the metrics in terms of weeks since you seem interested in that unit of metric (instead of 7, 14, 21 days --> 1, 2, 3 weeks)